woo-razorpay.php

```diff
@@ -2150,6 +2150,10 @@ public function update1ccOrderWC(& $order, $wcOrderId, $razorpayPaymentId)
                 if($razorpayData['shipping_fee'] == 0)
                 {
                     $item->set_method_title( 'Free Shipping' );
+                    if(isset($razorpayData['notes']) && isset($razorpayData['notes']['shipping_method_name']))
+                    {
+                        $item->set_method_title($razorpayData['notes']['shipping_method_name']);
```

Comment:
Can we add a default shipping name if that key is not available? Like bellow
`$shippingMethodTitle = $razorpayData['notes']['shipping_method_name'] ?? 'Standard Shipping';
$item->set_method_title($shippingMethodTitle);`

Reply:
@nkttiwari what happens if shipping_method_name is empty?